pkg/sfu/codecmunger/vp8.go

```diff
@@ -1,4 +1,4 @@
-package sfu
+package codecmunger
```

Comment:
No functional change in this module. Just moving the existing `VP8Munger` -> `codecmunger.VP8`